src/core/regimes.rkt

```diff
@@ -42,6 +42,8 @@
   (define err-lsts* (flip-lists err-lsts))
   (timeline-push! 'baseline (apply min (map errors-score err-lsts*)))
   (timeline-push! 'accuracy (errors-score (option-errors best)))
+  (define repr (context-repr ctx))
+  (timeline-push! 'repr (format "~a" (representation-name repr)))
```

Comment:
You can also use `(~a <expr>)` which is equivalent to `(format "~a" <expr>)`. Much cleaner, but not the end of the world

Reply:
Done.